content/SCALE/SCALETutorials/Network/ConfigureIPv6.md

```diff
@@ -0,0 +1,96 @@
+IPv4 networks cannot see or communicate with an IPv6 website or network unless a gateway or some other implementation is configured to allow it.
+See [Implementing IPv6](https://www.truenas.com/docs/references/IPv6/) for more information.
+
+## Configuring IPv6 Addresses
+After configuring your network infrastructure for IPv6, assign the IP addresses for your SCALE system.
+Use the SCALE UI to configure your network settings.
+If setting SCALE up for the first time after a clean install, use the Console Setup menu to enter IPv6 addresses.
+
+### Configuring an Interface Using the Console Setup Menu
+If configuring your network settings using the Console Setup menu for the first time after installing SCALE, first configure the interface address.
+Type 1, then press <kbd>Enter</kbd>.
+Enter **eno1** in **name**, then the IPv6 address in **aliases**.
+
+{{< trueimage src="/images/SCALE/Network/ConfigureIPv6InterfaceWithConsoleSetupMenu.png" alt="SCALE Console Setup Menu Edit Interface" id="SCALE Console Setup Menu Edit Interface" >}}
+
+Save, then select **a** to apply and **p** to make it persist. Type **q** to return to the Console Setup menu.
+
+Next, configure the IIPv6 gateway address, and the nameserver addresses. Type **2**, then press <kbd>Enter</kbd>.
+Enter the name server addresses provided by your IT department or Internet Service Provider (ISP), and then the gateways.
+Save, then select **a** to apply and **p** to make it persist. 
+
+### Adding an IPv6 Interface in the UI
+Navigate to the **Network** screen to enter your network settings.
+
+Click on **Add** in the **Interfaces** to open the **Add Interface** screen.
+
+1. Enter **eno1** as the name for the interface if it is the primary interface.
+2. Clear the **DHCP** checkbox, then select **Autoconfigure IPv6** if you want to create the IP address using SLAAC.
+   This automatically configures the IPv6 address.
+   You can only use this option one time to configure an IPv6 address for the system.
+3. Enter the IPv6 address assigned to the NIC port if using a fixed IP address assignment.
+4. Click **Save**
+5. Test the change.
+   If adding the primary interface test the network connection by opening a new browser window.
+   Enter the IPv6 address inside square brackets in the URL address field, for example, [*ipv6 address*]. 
+   After the system comes up, save the changes to the network interface.
+
+To access the UI after configuring an IPv6 address, enter the IPv6 address inside square brackets in the browser URL field.
+You cannot access the UI with the assigned host name when the system is configured on an IPv6 network.
+
+### Connecting to the UI IPv6 Address
+Unlike IPv4, you must enter the IPv6 address with a square bracket preceding and following the address.
+You cannot enter the host name assigned to the SCALE system to access the UI.
+For example, enter <code>[<i>ffff-ff-59f8-100--12</i>]</code> into the URL field of the browser window to access the UI.
```

Comment:
I think this should be ffff:ff:59f8:100::12   (colons not dashes)

Reply:
actually in Windows you must use the dashes not the colons or it fails.

Reply:
Oops, this was a copy paste mistake, this example did need the colons so I could not use the same example from the SMB share. It is fixed now.

Reply:
Ahh, I never knew that the SMB variation required dashes instead of colons.  Always nice to learn new things.  :-)